networkd/network_server.ml

```diff
@@ -474,12 +474,96 @@ module Bridge = struct
 			| Bridge -> Sysfs.get_all_bridges ()
 		) ()
 
+	let destroy _ dbg ?(force=false) ~name () =
```

Comment:
I think that this function is doing perhaps a bit too much for the purpose of `destroy_existing_vlan_bridge`. I don't think we should touch the interfaces, for example. So much so, that we might just make the low-level calls from that function directly. E.g. just `remove_config name ` and `Ovs.destroy_bridge name` or `Brctl.destroy_bridge` (not sure if you need to remove the ports first in the Linux bridge case).